client/src/Component/SMSModal.js

```diff
@@ -0,0 +1,50 @@
+/* eslint-disable jsx-a11y/label-has-for */
+import { useState } from "react";
+import Modal from "react-bootstrap/Modal";
+import Button from "react-bootstrap/Button";
+
+function SMSModal(props) {
+    const [show, setShow] = useState(false);
+    const [disabled, setDisabled] = useState(false);
+    const handleClose = () => setShow(false);
```

Comment:
`setShow(false)` closes the modal, so we could give this function a more meaningful name of `hideModal`